web/src/components/Field.tsx

```diff
@@ -62,6 +56,15 @@ const FieldContainer = styled.div<FieldContainerProps>`
     `};
 `;
 
+const LinkContainer = styled.div`
+  display: flex;
+  justify-content: end;
```

Comment:
the correct form is `justify-content: flex-end`, but anyways this is not doing anything i think? we can just delete it & leave it as const ` LinkContainer = styled.div`` ` wdyt